src/models/team.js

```diff
@@ -1,4 +1,10 @@
 const { Model } = require('sequelize');
+var en = require('nanoid-good/locale/en');
+var customAlphabet = require('nanoid-good').customAlphabet(en);
+
+// Based on documentation here https://github.com/ai/nanoid
+const alphabet = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
i think we should keep it to letters only, thoughts?

Reply:
Shouldn't be an issue collision-wise, as long as the user base is small enough the probabilities are still in our favour. I'll make this change.